### PR TITLE
Fix performance features deploy

### PR DESCRIPTION
**Pull request by cynepco3hahue**
# Description

- make sure that rt-kernel script does not exit with the error code
- apply kubelet config as part of worker machineconfigpool, see https://bugzilla.redhat.com/show_bug.cgi?id=1777150
- update the README, with the correct role label for the node

Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

Fixes # (issue)

## Type of change

Please select the appropriate options:

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- tested locally

**Test Configuration**:

- Versions:
- Hardware:

## Checklist

- [X] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not being merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

File: features/performance/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ All manifests should be applied automatically on a new cluster via `make deploy`
 
 Before you apply any feature, you should:
 
-- add `worker-rt` role to relevant nodes `oc label node <node_name> machineconfiguration.openshift.io/role=worker-rt`
+- add `worker-rt` role to relevant nodes `oc label node <node_name> node-role.kubernetes.io/worker-rt=`
```
Comment (dshchedr) [inline features/performance/README.md]: Just to clarify: should we add roles manually? I don't see it in deploying scripts.

Comment (cynepco3hahue) [reply features/performance/README.md]: Yes, we had some discussion regarding it, and the conclusion was that possible situation when you want to apply `worker-rt` label only on a subset of nodes, so we decide to leave as a manual step.

**Pull request merged**